composer.json

```diff
@@ -16,7 +16,7 @@
         "symfony/framework-bundle": "^2.8|^3.0",
         "symfony-cmf/core-bundle": "^2.0",
         "doctrine/phpcr-bundle": "^1.1",
-        "doctrine/phpcr-odm": "^1.4|^2.0",
+        "doctrine/phpcr-odm": "^1.4.2",
```

Comment:
agreed. the chances that 2.0 won't work are small, but there could be something and then it would be annoying.